pep-0674.rst

```diff
@@ -231,58 +225,72 @@ losing support with older Python versions. The project provides a header
 file which provides ``Py_SET_REFCNT()``, ``Py_SET_TYPE()`` and
 ``Py_SET_SIZE()`` functions to Python 3.8 and older.
 
-PyTuple_GET_ITEM() and PyList_GET_ITEM()
-----------------------------------------
-
-The ``PyTuple_GET_ITEM()`` and ``PyList_GET_ITEM()`` macros are left
-unchanged.
-
-The code pattern ``&PyTuple_GET_ITEM(tuple, 0)`` and
-``&PyList_GET_ITEM(list, 0)`` is still commonly used to get access to
-the inner ``PyObject**`` array.
-
-Changing these macros is out of the scope of this PEP.
 
 
 Backwards Compatibility
 =======================
 
 The proposed C API changes are backward incompatible on purpose.
 
-At December 1, 2021, a code search on the PyPI top 5000 projects (4760
+At January 26, 2022, a code search on the PyPI top 5000 projects (4762
 projects in practice, others don't have a source archive) found that
-`only 14 projects are affected
-<https://bugs.python.org/issue45476#msg407456>`_ (0.3%):
+only 13 projects are affected (0.3%):
 
+* Cython (0.29.26)
+* PyGObject (3.42.0)
 * datatable (1.0.0)
-* frozendict (2.1.1)
 * guppy3 (3.1.2)
-* M2Crypto (0.38.0)
-* mecab-python3 (1.0.4)
-* mypy (0.910)
-* Naked (0.1.31)
+* numpy (1.22.1)
 * pickle5 (0.0.12)
+* psycopg2 (2.9.3)
+* pycurl (7.44.1)
 * pysha3 (1.0.2)
 * python-snappy (0.6.0)
-* recordclass (0.16.3)
+* recordclass (0.17.1)
 * scipy (1.7.3)
 * zodbpickle (2.2.0)
-* zstd (1.5.0.2)
 
-These 14 projects only use 4 macros as l-value:
+On these 13 projects, only 2 macros are used as l-value: ``Py_TYPE()``
+and ``Py_SIZE()``.
 
-* ``PyDescr_NAME()`` and ``PyDescr_TYPE()`` (2 projects)
-* ``Py_SIZE()`` (8 projects)
-* ``Py_TYPE()`` (4 projects)
-
-Moreover, `24 projects just have to regenerate their Cython code
-<https://bugs.python.org/issue45476#msg407416>`_ to use
-``Py_SET_TYPE()``.
+Moreover, 22 projects just have to regenerate their Cython code to use
```

Comment:
```suggestion
An additional 22 projects just have to regenerate their Cython code to use
```

As above, I'd suggest clarifying this as it confused me on my first reading of the PEP